patches/chrome-browser-prefs-browser_prefs.cc.patch

```diff
@@ -10,11 +10,11 @@ index b8232737dbf4dabada561044e522e44c906c53c8..cb55e2b38a40ae9141d30c5dfc27bc27
  #include "build/build_config.h"
  #include "chrome/browser/about_flags.h"
  #include "chrome/browser/accessibility/accessibility_ui.h"
-@@ -378,6 +379,7 @@ void RegisterProfilePrefsForMigration(
- 
- void RegisterLocalState(PrefRegistrySimple* registry) {
-   // Please keep this list alphabetized.
+@@ -513,6 +514,7 @@ void RegisterLocalState(PrefRegistrySimple* registry) {
+ #if defined(TOOLKIT_VIEWS)
+   RegisterBrowserViewLocalPrefs(registry);
+ #endif
 +  brave::RegisterLocalStatePrefs(registry);
```

Comment:
I moved this to the end of the list so we can override chrome registered pref values